### ISSUE TITLE
Challenging Process

### ISSUE THREAD

**Issue by eastrgn** [district0x/memefactory#552]
When Challenging meme, the challengers need to come with the greater version of the meme(+DANK = submitting entry) that he challenged. This will ensure that the challenge mechanism will be more constructive, healthier competition, build a reputation for the user and brings value to the meme/artwork itself. 
Both should not be able in  participate in voting process. Let the Community decide with voting which meme/artwork should win.
 by having a capped amount of DANK on voting should will help to bring more fairness in this situation. the person that hold a lot of DANK will not have any influence to decide the winner.By doing this numbers of voters will decide the winner and this also can rasie the activity in mf among the content creator and will open the door to new user as well.
so in order to counter the usage of DANK getting useless we can raise the cost to submit an entry.

i try my best to explain my idea. Maybe it will inspire you to get the better version of it.

**Comment by lasharor** [comment]
I don't think this is a good idea. Specifically since I don't participate in creating memes. This would leave people like me out of the challing process whilst there is no objective reason that non-creators could not participate. 

Additionally; capping the amount of DNT used in the voting process limits the overal worth of the token.

**Comment by eastrgn** [comment]
But that can be counter by charging them with certain amount of Dank which they will never get back. By this you have an option either you want to challenge with meme + voting which cost you effort + submition fees or you want to pay for vote.

By doing this maybe it can create a good ecosystem for meme factory for future.

**Comment by eastrgn** [comment]
Because if we look on the phase that we are right now,  we need more content creator. By having more content creator the activity will increase = to the numbers of content creator. Just my 2cent although im happy with the current system that we have.

**Comment by madvas** [comment]
What we're gonna implement: 
When meme gets challenge, creator will have option to cancel submission, so the meme doesn't have to go through vote and reveal periods if creator agrees with challenger that the submission is not good enough. This way creator can resubmit his correction faster. Such cancelled meme will show up in Resolved Challenges as normally memes do after voting. Users can reclaim votes there, in case they managed to vote before cancelling happened.

Into following area under the yellow button, will be new pink button "Cancel Submission", which will be shown only to creator. 
![image](https://user-images.githubusercontent.com/3857155/61593114-d1597880-abdb-11e9-90da-4279f8b7c92a.png)

Meme.sol smart-contract needs to be updated to have new cancel functionality. We should also be displaying status of such meme as "Cancelled".
